Java/main-sample/src/main/java/com/perfectomobile/sample/SimpleTest.java

```diff
@@ -0,0 +1,84 @@
+package com.perfectomobile.sample;
+
+import com.perfecto.reportium.client.ReportiumClient;
+import com.perfecto.reportium.client.ReportiumClientFactory;
+import com.perfecto.reportium.model.CustomField;
+import com.perfecto.reportium.model.Job;
+import com.perfecto.reportium.model.PerfectoExecutionContext;
+import com.perfecto.reportium.model.Project;
+import com.perfecto.reportium.test.TestContext;
+import com.perfecto.reportium.test.result.TestResult;
+import com.perfecto.reportium.test.result.TestResultFactory;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Created by yuvals on 7/18/2018.
+ */
+public class SimpleTest {
+
+
+    public static void main(String[] args) throws IOException {
+        //boolean test passed = true; // assume true until failure
+        TestResult testResult = TestResultFactory.createFailure("Test stop failure");// assume failure until proven passed
+        WebDriver driver = getDriver();
+        ReportiumClient reportiumClient = setReportingClient(driver);
+        try {
+            reportiumClient.testStart("simpleTest", new TestContext.Builder()
+                    .withTestExecutionTags("google")
+                    .withCustomFields(new CustomField("developer", "John"))
+                    .build());
+            reportiumClient.stepStart("browser navigate to google");
+            driver.get("http://www.google.com");
+            reportiumClient.stepEnd();
+            reportiumClient.stepStart("browser navigate to ebay");
+            driver.get("http://www.ebay.com");
+            reportiumClient.stepEnd();
+
+            //STOP TEST
+            testResult = TestResultFactory.createSuccess();
+
+            driver.close();
+        } catch (Exception e) {
+            e.printStackTrace();
+        } finally {
+            reportiumClient.testStop(testResult);
```

Comment:
dont do any operations after driver close.

Reply:
ok, fixed, can we merge i don't get what the conflict is about..